### PR TITLE
Automated release process

### PR DESCRIPTION
**Pull request by MetRonnie**
Requirements:
- [x] https://github.com/cylc/release-actions/pull/5
- [x] I have added myself to contributors
- [x] Need someone to add a PyPI token for cylc-sphinx-extensions to the repo as a secret

N.B. There isn't a robust way I'm aware of for automatically updating the copyright year in this repo as the README is in rst, which doesn't allow html tags (to put the year to be updated in a span)

### CODE REVIEW

**Comment by oliver-sanders** [comment]
Added `PYPI_TOKEN`

File: setup.py [hunk 1]
```diff
@@ -38,7 +38,8 @@ def get_version(filename):
 
 
 TESTS_REQUIRE = [
-    'pytest'
+    'pytest',
+    'wheel'
```
Comment (oliver-sanders) [inline setup.py:42]: ?

**Review by oliver-sanders — APPROVED** [review]
All good, one comment. @wxtim can you sanity check then we can test this with the new release.

File: setup.py (hunk 1 above)
Comment (MetRonnie) [reply setup.py:42]: Needed to build the package, as `wheel` is not installed by default on the GH runner. Alternatively, I could add `python3 -m pip install wheel` in the cylc/release-actions build action https://github.com/cylc/release-actions/blob/master/build-python-package/action.yml ?

**Review by wxtim — APPROVED** [review]
Seems sane. :)

**Pull request merged**